pkg/runner/local_docker.go

```diff
@@ -56,6 +56,8 @@ type LocalDockerRunnerConfig struct {
 	// Background avoids tailing the output of containers, and displaying it as
 	// log messages (default: false).
 	Background bool `toml:"background"`
+	// number of open files
+	Ulimits []string `toml: "ulimits"`
```

Comment:
I am somewhat torn about having the option as a slice -- by making this option into a slice rather than just a string, maybe csv, this makes the command-line flag pretty awkward. I would recommend just using the toml file to specify the toml array.